salt/utils/parsers.py

```diff
@@ -911,9 +911,17 @@ def check_running(self):
         Check if a pid file exists and if it is associated with
         a running process.
         '''
+        # There is no os.getppid method for windows
+        from salt.utils import is_windows
+        if is_windows():
+            from salt.utils.win_functions import get_parent_pid
```

Comment:
That's what I thought, but it wasn't resolving for me when I was doing it earlier. It is now though. I don't know what happened.